x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/hooks/form.tsx

```diff
@@ -262,7 +262,11 @@ export function useOnSubmit({
         setFormState('INVALID');
         return;
       }
-      if (agentCount !== 0 && !isAgentlessIntegration(packageInfo) && formState !== 'CONFIRM') {
+      if (
+        agentCount !== 0 &&
+        !(isAgentlessIntegration(packageInfo) || isAgentlessPackagePolicy(packagePolicy)) &&
```

Comment:
I noticed that in the previous PR I forgot to add `isAgentlessPackagePolicy(packagePolicy)`, fixing it now.